cmk/special_agents/agent_pure_storage_fa.py

```diff
@@ -21,7 +21,9 @@
 from cmk.special_agents.utils.argument_parsing import Args, create_default_argument_parser
 
 _LOGGER = logging.getLogger("agent_pure_storage_fa")
+__version__ = "1.0.0"
```

Comment:
The version should match the Checmk version that ships this agent, `"2.3.0b1"` in this case. It will be automatically updated to always match.